gnucash/import-export/import-main-matcher.c

```diff
@@ -90,7 +91,10 @@ enum downloaded_cols
 #define CSS_INT_PROB_REQUIRED_CLASS "intervention-probably-required"
 #define CSS_INT_NOT_REQUIRED_CLASS  "intervention-not-required"
 
-static QofLogModule log_module = GNC_MOD_IMPORT;
+/* Define log domain for extended debugging of matcher */
+#define G_MOD_IMPORT_MATCHER "gnc.import-export.main-matcher"
```

Comment:
Why define it, it is only used here, I may be wrong but why not just use...
static QofLogModule log_module = "gnc.import.main-matcher";
Also I think it should be just 'import'.

Reply:
I was just wanting debug information from the main-matcher and not from the whole import process. I have changed it back to gnc.import.main-matcher so that i can set just the matcher to provide debug information while running
One thing I am unsure of with the log domain definitions is whether if gnc.import and gnc.import.main-matcher were defined and the log-level for gnc.import was set as debug it would automatically include any subdomains at that log-level.  I will research thiswhen I have a bit of time

Reply:
> I was just wanting debug information from the main-matcher and not from the whole import process. I have changed it back to gnc.import.main-matcher so that i can set just the matcher to provide debug information while running
> One thing I am unsure of with the log domain definitions is whether if gnc.import and gnc.import.main-matcher were defined and the log-level for gnc.import was set as debug it would automatically include any subdomains at that log-level. I will research thiswhen I have a bit of time

I believe subdomains will inherit log levels from their parent domains.

Reply:
Thanks Geert, I thought that might be the case and was working on that assumption.